### ISSUE TITLE
musket

### ISSUE THREAD

**Issue by ATPGaming** [studio-minus/ppg-bugs-and-requests#1833]
**Feature description**
add a musket to the game that is like the flintlock

**Feature purpose**
roleplay I guess

**Comment by DindinYT37** [comment]
Isn't the musket already in the game?

**Comment by ATPGaming** [comment]
no that's the m1 garand

**Comment by DindinYT37** [comment]
> no that's the m1 garand

Not what I was talking about.

**Comment by ATPGaming** [comment]
flintlock is kinda like a shorter musket but thats about it

**Comment by DindinYT37** [comment]
> flintlock is kinda like a shorter musket but thats about it

Still not what I meant, I have a Musket weapon in my weaponry category and since it had no mod-label I thought it might be vanilla. I guess it's modded, then.

**Comment by ATPGaming** [comment]
no i looked up musket and there isn't a musket in vanilla

**Comment by DindinYT37** [comment]
> no i looked up musket and there isn't a musket in vanilla

Yeah, that's what I just said. I said, *"I guess it's modded, then."* meaning I already realized that the musket I have is apparently modded and not in vanilla People Playground.

**Comment by Drogo-man** [comment]
there is a musket mod that looks EXTREMELY vanilla.
